src/translations/en_translations.json

```diff
@@ -227,7 +228,7 @@
   "languageShortValuesVietnamese": "Tiếng Việt",
   "feesTitle": "Payment types accepted",
   "feesValuesSelfPay": "Self pay",
-  "feesValuesMedicaid": "Medicaid",
+  "feesValuesMedicaid": "Health First Colorado (Colorado's Medicaid Program)",
```

Comment:
we have been manually un-doing this change until confirmation from BHA that this language is what we want to move forward with. Ideally, changes shouldn't go into the google sheet until they are finalized but alas...

Reply:
Good to know! Will revert.

Reply:
This is resolved.